index.html

```diff
@@ -16,6 +16,169 @@
+          Long Exposure Photography allows you to capture movement in a still
+          image or to allow for images to be taken in dark conditions.
+        </p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed porta
+          vestibulum velit, eu malesuada tortor posuere a. Donec bibendum luctus
+          risus, eu lobortis velit fringilla nec. Donec eu accumsan purus. Nulla
+          eu varius dui. Ut dui est, malesuada vitae vehicula sed, ultrices non
+          metus. Integer pretium diam nec arcu aliquam, vitae malesuada nisl
+          posuere. Suspendisse erat diam, mattis ac sem et, ullamcorper
+          condimentum lectus. Aliquam est diam, egestas sed tortor non, luctus
+          consequat arcu. Maecenas dapibus libero sapien, sed blandit arcu
+          porttitor quis. Lorem ipsum dolor sit amet, consectetur adipiscing
+          elit. Suspendisse nulla elit, tempus at consequat a, finibus eleifend
+          mauris. Curabitur et molestie enim, a condimentum neque. Nam neque mi,
+          egestas ut volutpat eget, condimentum at tortor. Phasellus mauris
+          mauris, mattis et nunc finibus, ullamcorper eleifend orci. Quisque
+          rhoncus ipsum dictum eros lobortis, vel luctus tellus sodales.
+        </p>
+        <p>
+          In ornare sapien non risus semper pulvinar. Proin accumsan ac orci sed
+          dignissim. Donec et odio hendrerit, consectetur tellus sit amet,
+          consectetur orci. Suspendisse ac vestibulum libero. Etiam id nulla
+          commodo, tincidunt odio ut, malesuada turpis. Nulla vel auctor tellus.
+          Morbi faucibus lorem scelerisque, feugiat enim ac, egestas massa. Ut
+          efficitur eu ex sed luctus. In tortor ex, posuere id risus eget,
+          lacinia venenatis dolor. Praesent ut magna id eros cursus lacinia.
+        </p>
+
+        <img
+          src="https://images.unsplash.com/photo-1533894101476-54906836ae6c?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1030&q=80"
+          alt="Long Exposure Image of car headlights at night"
+        />
+        <p>
+          Praesent posuere sed lacus id tristique. Nulla non ante mauris. Nulla
+          a ex ex. Aenean sagittis sit amet arcu id tristique. Nullam hendrerit
+          eget purus id sollicitudin. Sed elementum aliquam ipsum, id sodales
+          erat molestie at. Donec nunc quam, rutrum in mi vitae, mattis
+          ultricies dolor. Aenean volutpat et dui vel tempor. Interdum et
+          malesuada fames ac ante ipsum primis in faucibus. Sed et purus quis
+          ipsum vehicula consequat. Donec sed sem hendrerit, rhoncus lacus sit
+          amet, vulputate purus. Nullam euismod tincidunt nibh, pellentesque
+          ultrices nulla accumsan eget.
+        </p>
+        <p>Below is a link to a video describing the technique in detail:</p>
+        <a href="https://www.youtube.com/watch?v=aHKt4sxZdXs">Click Here</a>
```

Comment:
You might enjoy this article https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/